### ISSUE TITLE
23.04 TODO List

### ISSUE THREAD

**Issue by bmatherly** [mltframework/shotcut#1434]
#### Blockers
- [x] Motion Tracker region size crash: https://forum.shotcut.org/t/release-candidate-version-23-04-20/38515/21
- [x] Dan: Windows screen blanks or sleeps: #1435 
- [x] Brian: Move track is broken: https://github.com/mltframework/shotcut/issues/1429
- [x] Dan: Color range is broken: https://forum.shotcut.org/t/23-04-color-range-regression/38609

#### Non-blocker
- [x] #1439
- [x] Timeline Right Click: https://forum.shotcut.org/t/release-candidate-version-23-04-20/38515/28
- [x] #1440
- [x] Glaxnimate preview does not update automatically; you need to move the mouse over the canvas for it to refresh.,

**Comment by ddennedy** [comment]
The WebM alpha export presets were fixed, but Ut Video alpha appears to be a bug in FFmpeg.

I have not been able to fix the video mode crash on the count generator with GPU effects turned on. That only occurs with the combination of GPU effects, counter, the playhead on the counter, and changing video mode. That is a very narrow set of conditions.

**Comment by bmatherly** [comment]
> Ut Video alpha appears to be a bug in FFmpeg.

So what then? Revert back to an older FFmpeg? Or maybe just add it as a known bug to the release notes.

> That is a very narrow set of conditions.

Maybe it is a reliable way to recreate an otherwise infrequent crash. But I agree - not blocking for this release. I captured the crash in GDB and inspected the backtrace. A solution did not immediately occur to me. I have bookmarked that forum post and I will probably cycle back to it at some point - unless you beat me to it.
